README.md

```diff
@@ -0,0 +1,28 @@
+# Overview
+
+Classroom seating is a web application empowering teachers to generate classroom seating plans.
+
+## Meta
+- [Homepage](https://github.com/Go-Team-Gamma/classroom-seating)
+- [Wiki](https://github.com/Go-Team-Gamma/classroom-seating/wiki)
+- [Trello Board](https://trello.com/b/pqQOUld5/classroom-seating)
+- [Team Chat](https://nextcloud.groovestomp.com/call/u9dksneb)
+
+## Dependencies
+- [Go](https://golang.org/)
+
+## Setup
```

Comment:
I think it will be even easier if we wrap these in a Makefile so we don't have to remember order of execution (I know it's golang basic, just want to make it as easy as possible haha)

Reply:
I'll leave that as an exercise for you. :-)